runtime/astar/src/lib.rs

```diff
@@ -350,6 +363,15 @@ impl pallet_dapp_staking_v3::BenchmarkHelper<SmartContract<AccountId>, AccountId
             .expect("Must succeed in test/benchmark environment.");
     }
 }
+#[cfg(feature = "runtime-benchmarks")]
+impl<SC, ACC> orml_oracle::BenchmarkHelper<CurrencyId, FixedU128, ConstU32<2>>
+    for BenchmarkHelper<SC, ACC>
+{
+    fn get_currency_id_value_pairs() -> sp_runtime::BoundedVec<(CurrencyId, FixedU128), ConstU32<2>>
+    {
+        sp_runtime::BoundedVec::default()
+    }
+}
```

Comment:
this is bounded so MaxEncodeSize should work. We're not running it anyway

Reply:
I meant the benchmark logic won't work - we won't be able to generate weights.

Reply:
I had the impression we're going to use our benchmarks

Reply:
I only added those because at the time when we integrated `orml-oracle`, it didn't have benchmark support.

Hence I made this custom pallet. I also added support for it to orml repo, and now we have access to it. Our custom pallet seems redundant.

Anyways, not to complicate things further, we can also clean it up later.